api-reference/20 Data Visualization Widgets/dxChart/1 Configuration/valueAxis/visualRangeUpdateMode.md

```diff
@@ -19,8 +19,11 @@ The visual range does not change.
 
 - *"auto"*      
 When the visual range is specified, the applied mode changes depending on the argument axis' [visualRangeUpdateMode](/api-reference/20%20Data%20Visualization%20Widgets/dxChart/1%20Configuration/argumentAxis/visualRangeUpdateMode.md '/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/argumentAxis/#visualRangeUpdateMode') at the moment when data is updated:
-    - if *"reset"* or *"shift"*, then *"reset"*;
+    - if *"reset"*, then *"reset"*;
     - if *"keep"*, then *"keep"*.
```

Comment:
This means that the value axis' mode is the same as the argument axis' mode. These ifs are unnecessary.